files/50unattended-upgrades

```diff
@@ -1,4 +1,3 @@
-/etc/apt/apt.conf.d/50unattended-upgrades 
 // Automatically upgrade packages from these (origin:archive) pairs
 Unattended-Upgrade::Allowed-Origins {
 	"${distro_id}:${distro_codename}-security";
```

Comment:
@sublimino : It was explained to me that the pentesters specifically said only update the security repo - @sirv1ne is this correct?